BadPotato1007.md

```diff
@@ -0,0 +1,65 @@
+---
+name: "@BadPotato1007"
+project: "Drumbox looper midi"
+---
+
+# RasberryPi Drumbox looper midi with advanced features
+
+## Summary
+
+The plan is to make a advanced MIDI device with rasberryPi, which will include a Drumbox, a looper, and a midi launchpad, featuring a 7" touch display, and a sensitive mic, topped off with a good quality mono spear for audio out. 
+
+There will be 2 potentiometers for volume control, 2 rotary encoders for mode selection and drum sound selection, 16 capacitive switches for the drum pad, and 16 RGB leds for the drum pad.
+You can select between 4 modes, which are the drumbox, the looper, the midi launchpad, and the settings mode. In the drumbox mode, you can play the drum pad and the mic will pick up the sound and play it back through the speaker. In the looper mode, you can record and play back loops. In the midi launchpad mode, you can play the drum pad and it will play midi notes. In the settings mode, you can change the volume, the mode, and the drum sound. When you change the mode, the drum sound will change to the default drum sound for that mode. All sounds and recordings will be saved on the device, and will be uploaded to my cloud server with 1TB storage to access them on any other device. Turning the rotary encoders will cycle between the different drum kits. Turning the potentiometers will change the volume and the pitches to calibrate the device. Using a simple button on the display, you can Turn on the screen saver kind of feature, during which the screen will display a equalizer that reacts to surrounding sounds and the leds will turn off, but the device will still be on and you can still play the drum pad. The device will be powered by a 12V 2A power supply. Witch will be stepped down to 5v for the rasberryPi. The dimension of the device will be 12" x 10" x 2.5" and will be made out of black acrylic sheets and some 3d printed parts. It will be a portable device that can be used anywhere. It will be a great device for musicians and producers alike, and also help me in my musical endeavours.
```

Comment:
```suggestion
You can select between 4 modes, which are the drumbox, the looper, the midi launchpad, and the settings mode. In the drumbox mode, you can play the drum pad and the mic will pick up the sound and play it back through the speaker. In the looper mode, you can record and play back loops. In the midi launchpad mode, you can play the drum pad and it will play midi notes. In the settings mode, you can change the volume, the mode, and the drum sound. When you change the mode, the drum sound will change to the default drum sound for that mode. All sounds and recordings will be saved on the device, and will be uploaded to my cloud server with 1TB storage to access them on any other device. Turning the rotary encoders will cycle between the different drum kits. Turning the potentiometers will change the volume and the pitches to calibrate the device. Using a simple button on the display, you can Turn on the screen saver kind of feature, during which the screen will display a equalizer that reacts to surrounding sounds and the leds will turn off, but the device will still be on and you can still play the drum pad. The device will be powered by a 12V 2A power supply, which will be stepped down to 5v for the rasberryPi. The dimension of the device will be 12" x 10" x 2.5" and will be made out of black acrylic sheets and some 3d printed parts. It will be a portable device that can be used anywhere. It will be a great device for musicians and producers alike, and also help me in my musical endeavours.
```